src/app/common/interfaces/models.ts

```diff
@@ -0,0 +1,74 @@
+export interface User {
```

Comment:
why this file? Use this one: https://github.com/mainflux/ui/blob/master/src/app/common/interfaces/mainflux.interface.ts